packages/alchemy/src/react/components/auth/card/passkey-added.tsx

```diff
@@ -0,0 +1,18 @@
+import { PasskeyIllustration } from "../../../icons/passkey.js";
+import { PoweredBy } from "../../poweredby.js";
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+export function PasskeyAdded() {
+  return (
+    <div className="flex flex-col gap-5 items-center">
+      <div className="flex flex-col items-center justify-center h-12 w-12">
+        <PasskeyIllustration />
+      </div>
+      <h3 className="font-semibold text-lg">Passkey added!</h3>
+      <p className="text-fg-secondary text-sm text-center">
+        You can use this passkey to sign in next time.
```

Comment:
flag that you'll want to pull these out into the strings file that @bswags is working on right now.

Reply:
good call!

Reply:
good catch! added!